hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -704,6 +704,16 @@ public void testShQuota() throws Exception {
     assertEquals(100, objectStore.getVolume("vol4")
         .getBucket("buck4").getQuotaInNamespace());
 
+    // test whether supports default quota unit as bytes.
+    String[] bucketArgs5 = new String[]{"bucket", "setquota", "vol4/buck4",
+        "--space-quota", "500"};
+    execute(ozoneShell, bucketArgs5);
+    out.reset();
+    assertEquals(500, objectStore.getVolume("vol4")
+        .getBucket("buck4").getQuotaInBytes());
+    assertEquals(100, objectStore.getVolume("vol4")
+        .getBucket("buck4").getQuotaInNamespace());
+
```

Comment:
Isn't this same as L704? If so, I don't think we changed anything on the namespace quota side, so no need of this?

Reply:
O yes. That's more like a extra check to see the namespace quota remains the same. But removing it won't hurt anything.

Reply:
Removed the namespace check as it is not relevant